src/cppcache/include/geode/DataOutput.hpp

```diff
@@ -378,9 +378,8 @@ class CPPCACHE_EXPORT DataOutput {
      */
   inline void writeFullUTF(const char* value, uint32_t length = 0) {
```

Comment:
If we are being sticklers and consistent this method has no business here. It is only used by the Message writeStringPart and should really be implemented there.